### PR TITLE
Add/organize introductions to guidelines chapters

### PR DESCRIPTION
**Pull request by DelazJ**
Some matters that are worth explaining at the beginning were placed later in the reading, which would not help a newcomer to fully understand. This PR thus moves them and adds/updates intro where missing.

Ticket: #

- [x] Backport to LTR documentation is required

### Minimal requirements for merging *(for maintainers)*

- [ ] The description of this PR is coherent with the manual and does not provide wrong information.
- [ ] This PR passes the checks.

### CODE REVIEW

File: source/docs/documentation_guidelines/index.rst [hunk 1]
```diff
@@ -4,6 +4,29 @@
...
+https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html.
+
+The following chapters will guide you through learning:
+
+* how to manage the documentation source files using `git <https://git-scm.com/>`_
+  system and the `GitHub <https://github.com/>`_ platform on which they are stored
+* how to modify the texts, provide screenshots... in a compliant way
+* how to share and ensure your modifications are pushed to the official docs.
+
+If you are looking for general rules on contributing to QGIS project or managing
```
Comment (havatv) [inline source/docs/documentation_guidelines/index.rst]:
```suggestion
If you are looking for general information about how to contribute to the QGIS project or manage
```

File: source/docs/documentation_guidelines/writing.rst [hunk 2]
```diff
@@ -7,27 +7,10 @@ Writing Guidelines
...
-the `Python documentation style guidelines
-<https://devguide.python.org/documenting/>`_.
-Below you will find some general guidelines for writing QGIS documentation
-using reST.
-
-If you are looking for general rules on contributing to QGIS project or managing
-repositories, you may find help at
-`Get Involved in the QGIS Community <https://qgis.org/en/site/getinvolved/index.html>`_.
+the `Python documentation style guidelines <https://devguide.python.org/documenting/>`_.
+For convenience, we provide below a set of general rules we rely on for writing
```
Comment (havatv) [inline source/docs/documentation_guidelines/writing.rst]:
```suggestion
For convenience, we provide a set of general rules we rely on for writing
```

File: source/docs/documentation_guidelines/writing.rst [hunk 3]
```diff
@@ -7,27 +7,10 @@ Writing Guidelines
...
+QGIS documentation.
```
Comment (havatv) [inline source/docs/documentation_guidelines/writing.rst]:
```suggestion
QGIS documentation below.
```

File: source/docs/documentation_guidelines/first_contribution.rst [hunk 4]
```diff
@@ -24,6 +24,15 @@ are trying to fix or write to the `QGIS-community-team list
 
 Let's now dive into the process.
 
+Documentation sources are stored using a git version control system and
```
Comment (havatv) [inline source/docs/documentation_guidelines/first_contribution.rst]:
```suggestion
Documentation sources are stored using the git version control system and are
```

File: source/docs/documentation_guidelines/first_contribution.rst [hunk 5]
```diff
@@ -24,6 +24,15 @@ are trying to fix or write to the `QGIS-community-team list
...
+available on GitHub at https://github.com/qgis/QGIS-Documentation.
+You may find two main ways, not mutually exclusive, to modify the files:
```
Comment (havatv) [inline source/docs/documentation_guidelines/first_contribution.rst]:
```suggestion
There are two main ways, not mutually exclusive, to modify the files:
```

File: source/docs/documentation_guidelines/first_contribution.rst [hunk 6]
```diff
@@ -13,7 +13,7 @@
 
 If you are reading these lines, it is certainly because you are willing
 to contribute to writing QGIS documentation and are looking for a how-to.
-Then you are at the right place. The current document will guide you through
+You are at the right place! The current document will guide you through
```
Comment (havatv) [inline source/docs/documentation_guidelines/first_contribution.rst]:
```suggestion
You have come to the right place! The current document will guide you through
```

File: source/docs/documentation_guidelines/index.rst (hunk 1 above)
Comment (DelazJ) [reply source/docs/documentation_guidelines/index.rst]: Actually, I'd remove the mention of repository management here, focusing only on "you need help, contact the list"

Comment (DelazJ) [reply source/docs/documentation_guidelines/index.rst]:
```suggestion
If you are looking for general information about how to contribute to the QGIS
```

File: source/docs/documentation_guidelines/index.rst [hunk 7]
```diff
@@ -4,6 +4,29 @@
...
+repositories, you may find help at `Get Involved in the QGIS Community
```
Comment (DelazJ) [inline source/docs/documentation_guidelines/index.rst]:
```suggestion
project, you may find help at `Get Involved in the QGIS Community
```

**Pull request merged**